estimation_tracker/tracker/templates/home.html

```diff
@@ -0,0 +1,38 @@
+{% load custom_tags %}
+
+<a href={% url 'create_task' %}>Create Task</a>
+
+
+{% if tasks %}
```

Comment:
If there is no tasks, could we display a message for the user?

Reply:
sure